vm2/framework/starcoin-framework/doc/treasury_fa.md

```
@@ -0,0 +1,986 @@
+-  [Function `destroy_withdraw_capability`](#0x1_treasury_fa_destroy_withdraw_capability)
+-  [Function `add_linear_withdraw_capability`](#0x1_treasury_fa_add_linear_withdraw_capability)
+-  [Function `remove_linear_withdraw_capability`](#0x1_treasury_fa_remove_linear_withdraw_capability)
+-  [Function `destroy_linear_withdraw_capability`](#0x1_treasury_fa_destroy_linear_withdraw_capability)
+-  [Function `is_empty_linear_withdraw_capability`](#0x1_treasury_fa_is_empty_linear_withdraw_capability)
+-  [Function `get_linear_withdraw_capability_total`](#0x1_treasury_fa_get_linear_withdraw_capability_total)
+-  [Function `get_linear_withdraw_capability_withdraw`](#0x1_treasury_fa_get_linear_withdraw_capability_withdraw)
+-  [Function `get_linear_withdraw_capability_period`](#0x1_treasury_fa_get_linear_withdraw_capability_period)
+-  [Function `get_linear_withdraw_capability_start_time`](#0x1_treasury_fa_get_linear_withdraw_capability_start_time)
+
+
+<pre><code><b>use</b> <a href="account.md#0x1_account">0x1::account</a>;
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)